Rakefile

```diff
@@ -11,17 +11,25 @@ def brew_install(package, *args)
     # brew did not error out, verify tmux is greater than 1.8
     # e.g. brew_tmux_query = 'tmux 1.9a'
     installed_version = versions.split(/\n/).first.split(' ')[1]
-    unless version_match?(options[:version], installed_version)
+    unless version_match?(options[:requires], installed_version)
       sh "brew upgrade #{package} #{args.join ' '}"
     end
   end
 end
 
-def version_match?(requirement, version)
+def version_match?(requirement, v)
   # This is a hack, but it lets us avoid a gem dep for version checking.
+  version = parse_version(v)
```

Comment:
Very weird! What's the value of `gem --version` on your machine? I can't reproduce what you're seeing.